builtins/amp-img.js

```diff
@@ -38,6 +38,20 @@ export class AmpImg extends BaseElement {
 
     /** @private {?../src/srcset.Srcset} */
     this.srcset_ = null;
+
+    /** @private {!Array<string>} */
+    this.attributesToPropagate_ = ['alt', 'referrerpolicy', 'aria-label',
+      'aria-describedby', 'aria-labelledby'];
+  }
+
+  /** @override */
+  attributeChangedCallback(name, unusedOldValue, unusedNewValue) {
+    if (name === 'src') {
```

Comment:
@dvoytenko What do you think is the right strategy to enforce cache image URLs here?